crates/client/db/src/lib.rs

```diff
@@ -242,6 +248,8 @@ impl Column {
             ContractToNonces => "contract_to_nonces",
             ContractClassHashes => "contract_class_hashes",
             ContractStorage => "contract_storage",
+            Messaging => "messaging",
+            MessagingNonce => "messaging_nonce",
```

Comment:
```suggestion
            L1Messaging => "l1_messaging",
            L1MessagingNonce => "l1_messaging_nonce",
```

will this be more clear?

Reply:
done